perf_bench_test.go

```diff
@@ -640,9 +641,9 @@ func calculateRPS(vmNum int) int {
 	return vmNum * baseRPS
 }
 
```

Comment:
please add a thorough description of this function and what it does

Reply:
Done